features/definitions_transformations.feature

```diff
@@ -31,17 +31,20 @@ Feature: Step Arguments Transformations
           Behat\Behat\Tester\Exception\PendingException;
       use Behat\Gherkin\Node\PyStringNode,
           Behat\Gherkin\Node\TableNode;
+      use Behat\Step\Given;
+      use Behat\Step\Then;
+      use Behat\Transformation\Transform;
 
       class FeatureContext implements Context
       {
           private $user;
 
-          /** @Transform /"([^\ "]+)(?: - (\d+))?" user/ */
+          #[Transform('/"([^\ "]+)(?: - (\d+))?" user/')]
```

Comment:
I'm not sure we should be changing this feature, instead we should rename it to `definitions_transformations_annotations.feature` (we already have `definitions_transformations_attributes.feature`)?